src/components/Card.tsx

```diff
@@ -10,7 +10,7 @@ export const InnerStepCard = styled.div`
   flex-direction: column;
   flex: 1 1 auto;
   min-height: 400px;
-  margin: 0 auto 5rem auto;
+  margin: 0 auto auto auto;
```

Comment:
I generally don't think we need a 5rem margin on the bottom here, but I'm worried about breaking a lot of stuff because of how old this style is, so can we override this from the preview-card style?